bucketeer/src/test/kotlin/io/bucketeer/sdk/android/internal/event/EventCreatorsKtTest.kt

```diff
@@ -0,0 +1,20 @@
+package io.bucketeer.sdk.android.internal.event
+
+import org.junit.Test
+
+class EventCreatorsKtTest {
+  @Test
+  fun testLabelTimeoutValueShouldInDoubleFormat() {
+    val input = 15000L
+    val expectedDouble = 15.0
+    val timeoutString = input.toStringInDoubleFormat()
+    assert(timeoutString == expectedDouble.toString())
```

Comment:
I didn't understand why you checked the string here since it's already being checked in the `assert(timeoutString == "15.0")`.

Reply:
let me remove it. 
It was my test to check against the previous code which used float